modules/hal_silabs/wiseconnect/CMakeLists.txt

```diff
@@ -123,7 +140,7 @@ endif() # CONFIG_BT_SILABS_SIWX91X
 if(CONFIG_WISECONNECT_NETWORK_STACK)
   zephyr_compile_definitions(
     SLI_SI91X_ENABLE_OS
-    SL_SI91X_SI917_RAM_MEM_CONFIG=1
+    SL_SI91X_SI917_RAM_MEM_CONFIG=2
```

Comment:
This configuration should use DTS values to set the correct value 1, 2 or 3, not hard-code one.